### PR TITLE
Feature/filter by checkbox

### PR DESCRIPTION
**Pull request by kammon10**
#### What's this PR do?
add filter by checkbox feature

#### Where should the reviewer start?
scripts.js line 13 filterByTags method

#### How should this be manually tested?
open whats cookin' and select recipes

#### Any background context you want to provide?
none

#### What are the relevant tickets?
none

#### Screenshots (if appropriate)
none

### CODE REVIEW

File: src/classes/RecipeRepository.js
Comment (anewb87) [inline src/classes/RecipeRepository.js:17]: Good catch. We don't want any repeats.

**Review by cagallo — APPROVED** [review]
Good to do this so we don't get duplicate recipes if they have a lot of tags.

File: src/index.html
Comment (anewb87) [inline src/index.html:31]: I'm thinking we can get rid of the type="checkbox" thing since we're technically not utilizing it. Might hold off a bit just in case.

File: src/scripts.js
Comment (anewb87) [inline src/scripts.js]: *Needs an edit. It won't work for us as-written.

File: src/scripts.js
Comment (anewb87) [inline src/scripts.js]: I think we meant to delete this before merging.

**Review by anewb87 — COMMENTED** [review]
Looks good. There are some things we need to delete.

**Review by cagallo — APPROVED** [review]
Good eye! Cleaning up the file!

**Review by anewb87 — APPROVED** [review]

File: src/classes/RecipeRepository.js
Comment (aliroemhildt) [reply src/classes/RecipeRepository.js:17]: yes, great catch y'all!

File: src/index.html
Comment (aliroemhildt) [reply src/index.html:31]: Lexy mentioned that we ended up not utilizing this, I haven't caught up to that point yet in the code reviews, but I really think that is the direction we need to go with these. I think it will hurt us on our score if we are using an input element in our html but are not actually capturing the input and rather just using it as a button. I will do some research into checkboxes and hopefully get a better understanding of how to best utilize them!

File: src/scripts.js
Comment (aliroemhildt) [inline src/scripts.js:24]: yes, from what I remember this also seems like the right direction! I think we want to select ALL checkboxes and see which ones are checked (aka have a value of true). I'm wondering if we need to find the id equivalent of querySelectAll though and select all elements with the checkbox class, instead of just selecting the entire section? 

sorry this is actually in reference to line 21, not 24

File: src/scripts.js
Comment (aliroemhildt) [inline src/scripts.js:40]: this looks awesome! Excited to take a look at it in action

File: src/index.html
Comment (aliroemhildt) [inline src/index.html:36]: curious if we use both of these classes or just 'tag'. We could consider deleting one if it is not getting used

**Pull request merged**
